### PR TITLE
Add bargrooves.live

### PR DESCRIPTION
**Pull request by Chezzer-ub**

### CODE REVIEW

File: bargrooves/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "url": [
+        "bargrooves.live",
+        "www.bargrooves.live"
+      ],
+    "description": {
+      "english": "Bargrooves curates bespoke soundtracks for discerning house music fans; compilations and playlists to suit your seasonal moods, featuring the best current dance music alongside well-loved classics. Use this extension to show your friends what your listening to."
+    },
+    "service": "Bargrooves",
+    "version": "0.9",
+    "logo": "https://bargrooves.live/_assets/_img/default.png",
```
Comment (Bas950) [inline bargrooves/dist/metadata.json]: Change this to imgur so its not from their CDN

File: bargrooves/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "author": {
+      "name": "Chezzer",
+      "id": "195979856733929472"
+    },
+    "url": [
+        "bargrooves.live",
+        "www.bargrooves.live"
+      ],
+    "description": {
+      "english": "Bargrooves curates bespoke soundtracks for discerning house music fans; compilations and playlists to suit your seasonal moods, featuring the best current dance music alongside well-loved classics. Use this extension to show your friends what your listening to."
```
Comment (Bas950) [inline bargrooves/dist/metadata.json]: "en" not "english"

File: bargrooves/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    "service": "Bargrooves",
+    "version": "0.9",
```
Comment (Bas950) [inline bargrooves/dist/metadata.json]: how can it be version .9 if this is first release?

File: bargrooves/dist/presence.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+var presence = new Presence({
+    clientId: "641955799869947914", //The client ID of the Application created at https://discordapp.com/developers/applications
+    mediaKeys: true //Enable use and detection of media key presses
+});
+
+presence.on("UpdateData", () => {
+    //UpdateData is always firing, and therefore should be used as your refresh cycle, or `tick`. This is called several times a second where possible.
```
Comment (Bas950) [inline bargrooves/dist/presence.js]: Remove the comments

File: bargrooves/dist/presence.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+presence.on("UpdateData", () => {
+    //UpdateData is always firing, and therefore should be used as your refresh cycle, or `tick`. This is called several times a second where possible.
+
+    //It is recommended to set up another function outside of this event function which will change variable values and do the heavy lifting if you call data from an API.
+    var presenceData = {
+        largeImageKey: "default", /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+        smallImageKey: "logo-square", /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+        smallImageText: "bargrooves.live", //The text which is displayed when hovering over the small image
+        largeImageText: "bargrooves.live",
```
Comment (Bas950) [inline bargrooves/dist/presence.js]: LargeImageText cannot be set.

File: bargrooves/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+  }
```
Comment (Bas950) [inline bargrooves/tsconfig.json]: Not needed since you dont have a presence.ts file.

File: bargrooves/dist/presence.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+});
+
+presence.on("UpdateData", () => {
+    //UpdateData is always firing, and therefore should be used as your refresh cycle, or `tick`. This is called several times a second where possible.
+
+    //It is recommended to set up another function outside of this event function which will change variable values and do the heavy lifting if you call data from an API.
+    var presenceData = {
+        largeImageKey: "default", /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+        smallImageKey: "logo-square", /*The key (file name) of the Large Image on the presence. These are uploaded and named in the Rich Presence section of your application, called Art Assets*/
+        smallImageText: "bargrooves.live", //The text which is displayed when hovering over the small image
```
Comment (Bas950) [inline bargrooves/dist/presence.js]: I suggest you use these images for small image keys: 
https://github.com/PreMiD/Presences/tree/master/.resources

**Comment by Chezzer-ub** [comment]
image

**Comment by Chezzer-ub** [comment]
changes fixed/added

**Comment by Bas950** [comment]
image

You forgot to change this.

**Comment by Chezzer-ub** [comment]
image
image
image
image

**Pull request merged**